plugins/paytovote/paytovote.go

```diff
@@ -0,0 +1,183 @@
+package paytovote
+
+import (
+	"fmt"
+
+	abci "github.com/tendermint/abci/types"
+	"github.com/tendermint/basecoin/types"
+	"github.com/tendermint/go-wire"
+)
+
+const (
+	TypeByteCreateIssue byte = 0x00
+	TypeByteVoteFor     byte = 0x01
+	TypeByteVoteAgainst byte = 0x02
+	TypeByteVoteSpoiled byte = 0x03
+)
+
+type P2VPluginState struct {
+	TotalCost    types.Coins
+	Issue        string
+	votesFor     int
+	votesAgainst int
+	votesSpoiled int
+}
+
+type P2VTx struct {
+	Valid            bool
+	Issue            string      //Issue being voted for
+	ActionTypeByte   byte        //How is the vote being cast
+	Cost2Vote        types.Coins //Cost to vote
+	Cost2CreateIssue types.Coins //Cost to create a new issue
```

Comment:
Merged to develop: https://github.com/tendermint/basecoin/commit/1ff535d8834b092be3c4a9a9b126714d232f2b1a